ui/src/app/views/workspace-edit.tsx

```diff
@@ -722,8 +773,27 @@ export const WorkspaceEdit = fp.flow(withRouteConfigData(), withCurrentWorkspace
             </div>
           </div>
         </WorkspaceEditSection>
-        <WorkspaceEditSection header='Request a review of your research purpose'
+        <WorkspaceEditSection header='Request a review of your research purpose for potential
+                                      stigmatization of Research Participants'
                               tooltip={toolTipText.reviewRequest}>
+          <Link onClick={() => this.setState({showStigmatizationDetails:
+              !this.state.showStigmatizationDetails})} style={{marginTop: '0.5rem'}}>
+            More info on stigmatization
+            {this.state.showStigmatizationDetails ? <ClrIcon shape='caret' dir='up'/> :
+              <ClrIcon shape='caret' dir='down'/>}
+          </Link>
+          {this.state.showStigmatizationDetails &&
+            <div>
+              <div style={styles.text}>
+                Populations that are historically medically underserved or underrepresented in
+                biomedical research are also more vulnerable to stigmatization. If your population
+                of interest includes the following categories defined as UBR by the All of Us
```

Comment:
text: UBR should be spelled out here - UBR in parens if desired

Reply:
karthik approved this; fixing